example.csv

```diff
@@ -0,0 +1,8 @@
+b,banana,20
+b,apple,100
+s,banana,100
+p,banana,13
+r,apple,10
+p,apple,20
+p,banana,5
+s,banana,50
```

Comment:
In checklist there is tip - Place the input and output files into the src/main/resources folder.
so all 3 files should be in the package